Added a note on how to get to the Microsoft offices from the airport (#15)
w3c/did-wg: Meetings/F2F/2020.01.Amsterdam.md

* **Meeting Time:** 8:30 am to 6 pm (08:30 - 18:00) each day (tentative)
* **Breaks:** 10:30-11am, 1-2pm, 3:30-4pm (tentative)

> Note on the site: although the Microsoft offices are not far from the main exit hall of the airport (which is also the exit hall if you take the train from downtown Amsterdam) there are currently major road works that would make the walk to the offices a challenge. Here is my advise: exiting the hall you get to, essentially, a bus station. You may want to take the bus 300 of the company R-Net, going to _the left if you come out of the hall_ in direction of Haarlem. Take one stop, getting off at the station “Handelskade”. From there you can find the offices: I have made a [zoomed map](https://tinyurl.com/udtx6hn) which also shows the bus stop. (The buses are large, red articulated buses, but there are other lines as well, not only the 300.)

Reviewer: ```suggestion
Note on the site: although the Microsoft offices are not far from the main exit of the airport (which is also the exit if you take the train from downtown Amsterdam), there is currently major road work that could make walking to the Microsoft offices a challenge. Here is my advice: After exiting the hall, you will see a bus station. You can take bus 300 of the company R-Net, going to _the left as you come out of the hall_, in the direction of Haarlem. Take the bus one stop, get off at the station “Handelskade”. From there you can find the offices: Here is a a [zoomed map](https://tinyurl.com/udtx6hn) which also shows the bus stop. (The buses are large, red articulated buses. There are other lines as well, so be sure to use only the 300.)
```
